gst/nnstreamer/tensor_merge/gsttensormerge.c

```diff
@@ -355,6 +356,8 @@ gst_tensor_merge_sink_event (GstCollectPads * pads, GstCollectData * data,
   switch (GST_EVENT_TYPE (event)) {
     case GST_EVENT_FLUSH_STOP:
       tensor_merge->need_segment = TRUE;
+      tensor_merge->need_set_time = TRUE;
+      gst_tensor_time_sync_flush (tensor_merge->collect);
```

Comment:
How about checking whether `tensor_merge->collect` is null

Reply:
I added new PR to check null param (not here, in public functions for collect pad)